bitnami/spark/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: v1
```

Comment:
why did you add this file? how does it work?

Reply:
I forgot to remove it, sorry.